src/index.ts

```diff
@@ -61,18 +61,24 @@ app.get('/review_runs', async (req, res) => {
   const baseUrl = environment.dev ? 'localhost:8080' : 'https://sm64.dev';
 
   for (const run of sm64Unverified) {
+    const rejectedRun = run;
     if (run.flags.filter(f => f.reject).length > 0) {
+      const rejectionFlags = run.flags.filter(f => f.reject).filter(f => f.title);
```

Comment:
This last filter just filters by flags that have a title. Don't they all have a title?

Reply:
also nit: can you put this line inside the `else` block where the run gets rejected